lib/models/per-bundle-addon-cache/index.js

```diff
@@ -68,21 +59,16 @@ class PerBundleAddonCache {
   constructor(project) {
     this.project = project;
 
-    // cache of the addon packageInfos for addons that are descendants of the project
-    // (excluding going into lazy engines). Lazy engines that refer to any of these
-    // will defer to the project-level one.
-    this.bundledProjectAddonPkgInfos = this._setupBundledProjectAddonPackageInfos();
-
     // The cache of bundle-host instances and their individual addon caches.
```

Comment:
Could you update this comment, since the cache is no longer a cache of instances but a cache of PackageInfos. Also, the 'empty string for a name' thing has gone away, right?